### PR TITLE
feat(checkout): CHECKOUT-5958 Add missing field from Storefront Consignment API

### PR DESCRIPTION
**Pull request by valentindellangela**
## Description:
Updating the public facing Storefront Consignment API docs adding the missing field `shouldSaveAddress` we noticed while working on https://jira.bigcommerce.com/browse/CHECKOUT-5940 (difference between S2S and Storefront API)

## Testing:
Swagger Editor

POST
image
image

PUT
image
image

Also in the response:
image

cc @bigcommerce/dev-docs @bigcommerce/checkout

### CODE REVIEW

File: reference/checkouts.sf.yml [hunk 1]
```diff
@@ -2563,6 +2568,9 @@ definitions:
           id:
             description: ''
             type: string
+          shouldSaveAddress:
+            description: 'Indicates if we should add this address to the customer address book'
```
Comment (traci707) [inline reference/checkouts.sf.yml:2572]: 'Indicates if we should add this address to the customer address book.'

File: reference/checkouts.sf.yml [hunk 2]
```diff
@@ -3546,6 +3554,9 @@ definitions:
                   type: string
                 fieldValue:
                   type: string
+          shouldSaveAddress:
+            description: 'Indicates if we should save this address in the customer address book'
```
Comment (traci707) [inline reference/checkouts.sf.yml:3558]: 'Indicates if we should save this address in the customer address book.'

File: reference/checkouts.sf.yml [hunk 3]
```diff
@@ -3635,6 +3646,9 @@ definitions:
                   type: string
                 fieldValue:
                   type: string
+          shouldSaveAddress:
+            description: Indicates if we should save this address in the customer address book
```
Comment (traci707) [inline reference/checkouts.sf.yml:3650]: 'Indicates if we should save this address in the customer address book.'

**Review by traci707 — COMMENTED** [review]
Added minor edits

File: reference/checkouts.sf.yml (hunk 1 above)
Comment (valentindellangela) [reply reference/checkouts.sf.yml]: Thanks, I updated the description

**Review by davidchin — APPROVED** [review]
Nice one

**Comment by valentindellangela** [comment]
@traci707 happy for you to merge this ticket whenever you want 🙇‍♂️ thanks

**Review by uwikaiddi — APPROVED** [review]

**Pull request merged**